x-pack/legacy/plugins/ml/public/data_frame/pages/job_management/components/job_list/preview_pane.tsx

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, useEffect, useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiInMemoryTable, EuiInMemoryTableProps, SortDirection } from '@elastic/eui';
+import { getFlattenedFields } from '../../../../components/source_index_preview/common';
+import { ml } from '../../../../../services/ml_api_service';
+
+import { DataFramePreviewRequest } from '../../../../common';
+
+interface Transform {
+  dest: { index: string };
+  id: string;
+  pivot: {
+    group_by: { [key: string]: any };
+    aggregations: { [key: string]: any };
+  };
+  source: {
+    index: string[];
+    query: { [key: string]: any };
+  };
+}
+
+interface TransformData {
```

Comment:
Suggestion to use a copy of what's in `get_jobs.ts` here:

```
import { DataFrameTransformWithId } from '../../../../../common';
interface GetDataFrameTransformsResponse {
  count: number;
  transforms: DataFrameTransformWithId[];
}
```

Then you don't need the `Transform` interface above at all. We can move the copies of `DataFrameTransformWithId` to a common shared location in a follow up (I plan to do that in the PR that will rename all `job` references to `transform`.

Reply:
Updated in: fd3c34164e5ead57a869c15608d253990f2a3f20